CMakeLists.txt

```diff
@@ -688,8 +688,14 @@ if(USE_FBGEMM AND ((CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND CMAKE_SIZEOF_VO
   set(USE_FBGEMM OFF)
 endif()
 
+set(BUILD_ONEDNN_GRAPH OFF)
+
 include(cmake/Dependencies.cmake)
 
+if(BUILD_ONEDNN_GRAPH)
+  string(APPEND CMAKE_CXX_FLAGS " -DBUILD_ONEDNN_GRAPH")
```

Comment:
Actually, this is less then ideal, as it passes this option to all libraries, rather than to just torch_cpu and torch_python

Reply:
Thanks for your advice, @malfet! I'd try to ensure that it'd only be passed to the libraries that use it.
However, the same thing holds true for other flags in `CMakeLists.txt` as well, for example, https://github.com/pytorch/pytorch/blob/master/CMakeLists.txt#L703-L742.

Reply:
@malfet, I've made this change. Thank you! :)
Can you please re-import the PR? Sorry for the inconvenience!